tests/core_tests/wallet_tools.cpp

```diff
@@ -0,0 +1,289 @@
+//
+// Created by Dusan Klinec on 2019-02-28.
+//
+
+#include "wallet_tools.h"
+#include <random>
+
+using namespace std;
+using namespace epee;
+using namespace crypto;
+using namespace cryptonote;
+
+// Shared random generator
+static std::random_device RD;
+static std::mt19937 MT(RD());
```

Comment:
Know this is test code, but this isn't a CSRNG. What do you think about using a [libsodium custom-rng](https://download.libsodium.org/doc/advanced/custom_rng), or Monero's RNG?

Might prevent accidental use of Mersenne-twister in production code.

Reply:
OK for peace of mind I've changed it to use `crypto::rand` and for shuffling `std::default_random_engine(crypto::rand<unsigned>())`, the same as in the production code (e.g., tx output shuffling).